1/debian-10/rootfs/opt/bitnami/scripts/mediawiki-env.sh

```diff
@@ -98,6 +100,8 @@ MEDIAWIKI_SMTP_USER="${MEDIAWIKI_SMTP_USER:-"${SMTP_USER:-}"}"
 export MEDIAWIKI_SMTP_USER="${MEDIAWIKI_SMTP_USER:-}" # only used during the first initialization
 MEDIAWIKI_SMTP_PASSWORD="${MEDIAWIKI_SMTP_PASSWORD:-"${SMTP_PASSWORD:-}"}"
 export MEDIAWIKI_SMTP_PASSWORD="${MEDIAWIKI_SMTP_PASSWORD:-}" # only used during the first initialization
+MEDIAWIKI_SMTP_AUTH="${MEDIAWIKI_SMTP_AUTH:-"${SMTP_AUTH:-}"}"
+export MEDIAWIKI_SMTP_AUTH="${MEDIAWIKI_SMTP_AUTH:-}" # only used during the first initialization
```

Comment:
In accordance with my above suggestion:

```suggestion
MEDIAWIKI_SMTP_AUTH="${MEDIAWIKI_ENABLE_SMTP_AUTH:-"${SMTP_AUTH:-}"}"
export MEDIAWIKI_SMTP_AUTH="${MEDIAWIKI_ENABLE_SMTP_AUTH:-}" # only used during the first initialization
```